foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiChatCompletionRequest.java

```diff
@@ -99,54 +110,151 @@ public OpenAiChatCompletionRequest(@Nonnull final String message) {
    * @param message the primary message to be added to the prompt
    * @param messages additional messages to be added to the prompt
    */
+  @Tolerate
   public OpenAiChatCompletionRequest(
       @Nonnull final OpenAiMessage message, @Nonnull final OpenAiMessage... messages) {
+    // Keeps default values for boolean fields. @With introduces bug comparison of Boolean
+    this(
+        new ArrayList<OpenAiMessage>(),
```

Comment:
(Minor)

Please use Guava for convenient + immutable list creation.

```suggestion
        Lists.asList(message, messages),,
```